invoicing/api/views.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+from rest_framework import generics
+
+from ..api.serializers import ProjectInstallmentTemplateSerializer
+from ..models import ProjectInstallmentTemplate
+
+
+class ProjectInstallmentAPIView(generics.ListCreateAPIView):
```

Comment:
Could be good to add a comment that this is a "bulk" creation view. Just for future developers so that they at a quick glance know that this is for creating and listing multiple installments, and not just for creating a single one.